### ISSUE TITLE
Fallback for non-HTML rendering of help

### ISSUE THREAD

**Issue by klmr** [noamross/htmlhelp#5]
What happens when the help is not rendered to HTML but rather to PDF or troff/text? If I am reading the code correctly, the contents of the tag will then just be echoed verbatim, which probably isn’t very useful. Does it make more sense to hide the content? Or maybe the document writer should be able to provide a fallback.

**Comment by noamross** [comment]
Yes, one currently must wrap macros in `if{html}{MACRO}`. Possibly, together with `if{latex}` or other `\ifelse`macros (See [this section of WRE](https://cran.rstudio.com/doc/manuals/r-release/R-exts.html#Conditional-text)].  I will probably wrap the existing macros with this, but have to think about how some of the nested macros will work.
